src/components/MainCarousel.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  useCallback,
+  useEffect,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from 'react'
+import ChevronLeft from 'icons/ChevronLeft'
+import ChevronRight from 'icons/ChevronRight'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  display,
+  flex,
+  flexDirection,
+  fontSize,
+  gap,
+  height,
+  justifyContent,
+  margin,
+  opacity,
+  overflow,
+  scrollSnap,
+  textAlign,
+  textColor,
+  width,
+} from 'classnames/tailwind'
+
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-6')
```

Comment:
Let's limit the height, so we don't encounter jumpy carousel

Reply:
Since it's fixed width, I fixed the font sizes instead so it shouldn't do this at any of the 3 widths.